openwisp_network_topology/tests/test_selenium.py

```diff
@@ -76,6 +76,12 @@ def setUp(self):
             organization=org,
         )
 
+    def tearDown(self):
+        # Workaround to clear the web driver console logs
+        # after the invocation of each test method
```

Comment:
I asked ChatGPT to shorten this:

Clear the web driver console logs after each test method.

I ended up tweaking it to:

Clear the web driver console logs after each test.

Less is more.